packages/scaffold-config/src/ct-detect-third-party.ts

```diff
@@ -180,17 +197,22 @@ export async function detectThirdPartyCTFrameworks (
           return !!validateThirdPartyModule(m)
         } catch (e) {
           debug('Failed to parse third party module with validation error: %o', e)
+          erroredFrameworks.push({
+            name,
+            path: modulePath,
+            reason: 'error while parsing',
+          })
```

Comment:
Wouldn't this reflect the state variables for the last module in the promise chain above, not necessarily the module being validated in this try/catch?

Reply:
I removed the use of these higher-scoped variables, now we only use values in the `catch` that were in scope above in the `try`. This should ever wrongly reporting error paths by grabbing a value that had been overwritten.